src/classes/AccountDeDuper.cls

```diff
@@ -41,7 +41,7 @@ Public List<Account> SearchAccount (Account pCurrentRecord, String pAccountName,
                     ListOfAdditionalMatches = [select id, Name, BillingStreet, BillingCity, BillingPostalCode, BillingState, Phone, Category__c, Owner.Alias, Record_Type_Name__c from Account where Name LIKE :AccountLikeName and BillingPostalCode=:pPostCode order by Name LIMIT :currentLimit ];
                 }
                 else {
-                    ListOfAdditionalMatches = [select id, Name, BillingStreet, BillingCity, BillingPostalCode, BillingState, Phone, Category__c, Owner.Alias, Record_Type_Name__c from Account where Name LIKE :AccountLikeName and BillingPostalCode=:pPostCode AND RecordTypeId =:pRecordTypeId order by Name LIMIT :currentLimit ];
+                    ListOfAdditionalMatches = [select id, Name, BillingStreet, BillingCity, BillingPostalCode, BillingState, Phone, Category__c, Owner.Alias, Record_Type_Name__c from Account where Name LIKE :AccountLikeName and BillingPostalCode=:pPostCode AND RecordTypeId =:pRecordTypeId order by Name ];
                 }
```

Comment:
* This method has a very long parameter list. I suggest to encapsulate many related parameters into a single object, or consider passing some parameters in the constructor prior to the invocation of this method.
* Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.